packages/client-utils/src/types.d.ts

```diff
@@ -34,4 +35,12 @@ export type TypedPublished<T extends string> = T extends keyof PublishedTypeMap
           ? OutcomeRecord
           : T extends `vaultFactory.managers.manager${number}.metrics`
             ? VaultManagerMetrics
-            : unknown;
+            : T extends 'agoricNames.instance'
+              ? Array<[string, Instance]>
+              : T extends 'agoricNames.brand'
+                ? Array<[string, Brand]>
+                : T extends 'fastUsdc'
+                  ? ContractRecord
+                  : T extends 'fastUsdc.poolMetrics'
```

Comment:
This coupling still seems odd to me.

To my mind, fast-usdc is in the agoric-sdk for logistical reasons; any 3rd party developer should be able to make a similar app. Do we expect 3rd-party developers to update client-utils for every vstorage path they introduce?

Is there any way to flip it around so that a dev can combine types from agoricNames, inter protocol, and fast-usdc? I did something like that once...

https://github.com/Agoric/agoric-sdk/blob/9418efc6d8ba27f3361754ea2a55b47a367e06b3/packages/boot/tools/supports.ts#L60-L66